blueocean-dashboard/src/main/js/components/Pipelines.jsx

```diff
@@ -10,8 +10,22 @@ const { array } = PropTypes;
 
 export default class Pipelines extends Component {
 
+    constructor(props) {
+        super(props);
+        this.state = {
+            showAllPipelines: false,
+        };
+        this.onShowAllClick = this.onShowAllClick.bind(this);
+    }
+
+    onShowAllClick() {
+        this.state.showAllPipelines = true;
+        this.forceUpdate();
+    }
+
```

Comment:
I know you're new to React: the best way to do this is to use the setState() method which triggers an "update" (which triggers `render()` again:

e.g. 
```
this.setState({
    showAllPipelines: true
});
```